docs/source/run-workflows/observe/observability-propagation.md

```
@@ -0,0 +1,318 @@
+  _type: fastapi
+  observability:
+    embed_trace_in_response: true
+```
+
+When enabled, the streaming response includes the `_trace` payload as the final event after all intermediate steps and output chunks.
+
+### Response Headers
+
+Non-streaming responses include the `Observability-Trace-Id` header:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)